egs/wsj/s5/steps/nnet3/chain/train.py

```diff
@@ -463,10 +467,10 @@ def TrainOneIteration(dir, iter, srand, egs_dir,
     TrainNewModels(dir, iter, srand, num_jobs, num_archives_processed, num_archives,
```

Comment:
I am suggesting that you use named arguments while calling the function and not change the function definition. 

We have been constantly updating the argument list for these function, so better change it to use named arguments to avoid user errors. Vimal and I have been modifying all the function calls with more than a few arguments to use named arguments so I would recommend that here too.